CHANGELOG.md

```diff
@@ -15,6 +15,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#2870](https://github.com/open-telemetry/opentelemetry-python/pull/2870))
 - Fix: Remove `LogEmitter.flush()` to align with OTel Log spec
   ([#2863](https://github.com/open-telemetry/opentelemetry-python/pull/2863))
+- Fix metric reader examples + added `preferred_temporality` and `preferred_aggregation`
```

Comment:
nit maybe this doesn't need to be in changelog since it is just a doc update

Reply:
I think it should since the `preferred_X` configurations were added to `ConsoleMetricsExporter` as well.